src/engine/controls/loopingcontrol.cpp

```diff
@@ -354,19 +354,29 @@ double LoopingControl::nextTrigger(bool reverse,
 
     LoopSamples loopSamples = m_loopSamples.getValue();
 
+    // adjust loop in was toggled (slotLoopIn or slotLoopOut)
     if (m_bAdjustingLoopInOld != m_bAdjustingLoopIn) {
         m_bAdjustingLoopInOld = m_bAdjustingLoopIn;
-        if (reverse && !m_bAdjustingLoopIn) {
+
+        // adjust loop in was true and is now false and is only relevant in reverse mode
+        // in non-quantized mode we jump immediately to the end of the loop while in quantized mode we wait until the loop start is reached
+        if (reverse && !m_bAdjustingLoopIn && !m_pQuantizeEnabled->toBool()) {
```

Comment:
// If the loop in button was released in reverse mode, we need to jump to loop end to not fall out of the just set loop. This must not happen in qantized mode. Here we need to jump to a qantized position, which is handeled in ....